mgmtsystem/security/mgmtsystem_security.xml

```diff
@@ -52,4 +52,89 @@
         <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
     </record>
 
+    <!-- Read/Write/Create/Unlink access in 'Management System / viewer' group
+        only for 'Management system' documents -->
+    <record id="document_page_rule_mgmtsystem_viewer" model="ir.rule">
+        <field name="name">Document page for viewers</field>
+        <field name="model_id" ref="document_page.model_document_page"/>
+        <field name="domain_force">['|', ('management_system', '=', True), ('parent_id.management_system', '=', True)]</field>
```

Comment:
What about a page marked as management_system=True that has a child page that has another child page? The 2nd child would be hidden? :thinking: 

Maybe we can use `child_of` somehow? Or then maybe it's better to remove `('parent_id.management_system', '=', True)`.